c2corg_ui/templates/account.html

```diff
@@ -38,6 +38,13 @@ <h1 translate>Change account parameters</h1>
             ng-minlength="3" maxlength="15" ng-pattern="/^(?!.*[^a-zA-Z0-9])/" class="form-control" />
         </div>
 
+        <div id="account-is_profile_public-group" class="form-group">
+          <label>
+            <input type="checkbox" name="is_profile_public" ng-model="account.is_profile_public" class="form-control" />
+            <span translate>Make profile page public</span>
```

Comment:
Maybe add a tooltip "Unauthenticated users can see your profile page" to make it clear.